lessons/Early-Math/absolute-value-and-opposites/langs/en-us/index.adoc

```diff
@@ -0,0 +1,150 @@
+= Absolute Value and Opposites
+
+@description{Students use Circles of Evaluation to analyze the effect of negation and absolute value.}
+
+@lesson-prereqs{distributive-property}
+
+@keywords{Opposite, Absolute Value}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Define absolute value as a number's distance from zero on the number line.
+* Recognize that two numbers are referred to as "opposites" if they are the same distance from zero.
+* Use Circles of Evaluation to analyze expressions with opposite and absolute value.
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Circles of Evaluation to think about opposites and absolute value.
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Preparation
+|
+* Draw or project a number line, which will be the focal point of a lot of the conversation for today's lesson.
```

Comment:
Let's be more specific! Of the integers from -10 to 10? or are you picturing something else?